Podfile

```diff
@@ -52,8 +52,8 @@ def wordpress_ui
 end
 
 def wordpress_kit
-    pod 'WordPressKit', '~> 2.1.0.1'
-    #pod 'WordPressKit', :git => 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', :commit => 'f2e1d41'
+    #pod 'WordPressKit', '~> 2.1.0.2'
+    pod 'WordPressKit', :git => 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', :commit => '9971cb763091604fde1b6efe5babf2c2772ba11d'
```

Comment:
Is this commit hash / change in WordPressKit important for this review? I didn't see anything in the summary.

Reply:
👋 @mindgraffiti - I just dropped in, as I'm evaluating the impact of the change from the `WordPressKit` side.

If it helps, that commit hash is referenced in this PR : https://github.com/wordpress-mobile/WordPressKit-iOS/pull/120

Reply:
Thanks, yes that's the one. I forgot to reference that PR in the description 🤦‍♂️

Reply:
Oh! That makes sense. Thanks @stevebaranski